From 3ebb88911c8b333848a89f46460c981202585846 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Sun, 6 Dec 2015 03:46:22 +0100
Subject: [PATCH 04/10] dicom in upstream now pydicom

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 pysegbase/dcmreaddata.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pysegbase/dcmreaddata.py b/pysegbase/dcmreaddata.py
index 5c7bd65..c23195f 100644
--- a/pysegbase/dcmreaddata.py
+++ b/pysegbase/dcmreaddata.py
@@ -11,7 +11,7 @@ $ dcmreaddata -d sample_data -o head.mat
 
 import sys
 import os
-import dicom
+import pydicom as dicom
 import numpy as np
 from optparse import OptionParser
 from scipy.io import savemat
-- 
2.6.3

