From bef64d1e3030dae0a296d233fd7e8ecf42155665 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Sun, 6 Dec 2015 03:54:37 +0100
Subject: [PATCH 07/10] py3: dict methods

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 pysegbase/seed_editor_qt.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pysegbase/seed_editor_qt.py b/pysegbase/seed_editor_qt.py
index 09757bc..0f8cd67 100644
--- a/pysegbase/seed_editor_qt.py
+++ b/pysegbase/seed_editor_qt.py
@@ -58,7 +58,7 @@ CONTOURS_COLORTABLE[:,:3] = 255
 CONTOURLINES_COLORTABLE = np.zeros((256,2,4), dtype=np.uint8)
 CONTOURLINES_COLORTABLE[:,:,:3] = 255
 
-for ii, jj in CONTOURS_COLORS.iteritems():
+for ii, jj in CONTOURS_COLORS.items():
     key = ii - 1
     CONTOURS_COLORTABLE[key,:3] = jj
     CONTOURS_COLORTABLE[key,3] = 64
@@ -786,7 +786,7 @@ class QTSeedEditor(QDialog):
         self.volume_unit = volume_unit
 
         self.last_view_position = {}
-        for jj, ii in enumerate(VIEW_TABLE.iterkeys()):
+        for jj, ii in enumerate(VIEW_TABLE.keys()):
             if viewPositions is None:
                 viewpos = img.shape[VIEW_TABLE[ii][-1]] / 2
 
@@ -1165,7 +1165,7 @@ class QTSeedEditor(QDialog):
 
     def cropUpdate(self, img):
 
-        for ii in VIEW_TABLE.iterkeys():
+        for ii in VIEW_TABLE.keys():
             self.last_view_position[ii] = 0
         self.actual_slice = 0
 
-- 
2.6.3

