From 5d2512dfe4298304199aa246ccbf058da29e4966 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Sun, 6 Dec 2015 04:00:10 +0100
Subject: [PATCH 09/10] py3: raw_input() -> input()

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 pysegbase/dcmreaddata.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pysegbase/dcmreaddata.py b/pysegbase/dcmreaddata.py
index 9f5e31f..0577764 100644
--- a/pysegbase/dcmreaddata.py
+++ b/pysegbase/dcmreaddata.py
@@ -143,7 +143,7 @@ class DicomReader():
                         print('series')
                         series_info = self.dcmdirstats()
                         print(self.print_series_info(series_info))
-                        snstring = raw_input('Select Serie: ')
+                        snstring = input('Select Serie: ')
 
                     sn = int(snstring)
                 else:
-- 
2.6.3

